lib/convection/model/template.rb

```diff
@@ -183,8 +183,75 @@ def diff(other, stack_ = nil)
       end
 
       def to_json(stack_ = nil, pretty = false)
-        return JSON.generate(render(stack_)) unless pretty
-        JSON.pretty_generate(render(stack_))
+        stack = render(stack_)
+
+        #bytesize
+        json = JSON.generate(stack)
+        template_bytesize = json.bytesize
+        cf_max_bytesize = 51200
```

Comment:
This should be defined as a constant in the scope of the class instead of a variable in the method